src/ios/VLCPlugin.m

```diff
@@ -687,6 +779,34 @@ - (void) vlc_clearHeartbeatTimer {
 
 #pragma mark Lock Screen Metadata
 
+- (NSDictionary *)lockScreenMetadataForAudio:(NYPRAudio *)audio {
+    NSString *title = [self titleForAudio:audio];
+    NSString *subtitle = [self subtitleForAudio:audio];
+    NSString *imageFileUrl = [audio.imageFileUrl absoluteString] ? [audio.imageFileUrl absoluteString] : @"";
+    NSDictionary *metadata = @{kVLCPluginAudioMetadataKeyTitle : title, kVLCPluginAudioMetadataKeyArtist : subtitle, kVLCPluginAudioMetadataKeyLockscreenImageUrl : imageFileUrl };
+    return metadata;
+}
```

Comment:
Include a nil check here for imageFileUrl and don't pass image key if no image exists:

    NSDictionary *metadata;
    
    if (imageFileUrl) {
        metadata = @{kVLCPluginAudioMetadataKeyTitle : title, kVLCPluginAudioMetadataKeyArtist : subtitle, kVLCPluginAudioMetadataKeyLockscreenImageUrl : imageFileUrl };
    }
    else {
        metadata = @{kVLCPluginAudioMetadataKeyTitle : title, kVLCPluginAudioMetadataKeyArtist : subtitle };
    }